New landing page for docs site (#857)
Agoric/documentation: main/README.md

# run `yarn docs:dev` at any time to start local dev server and access
# website at localhost:8080 by default
####

> home: false # use default home page layout (hero image with text, features section)

Reviewer: drop the comment? (or update it)

Author: Removed comment

footer: Apache-2.0 Licensed | Copyright © 2023 - Agoric
---

<div class="home-banner">
  Receive our latest tutorials, webinar invites, and much more! <a href="https://agoric.com/dev-newsletter" style="color: white;"">Join our newsletter</a>
</div>

<div class="home-section">
  <HomeButtonHeader
    title="Agoric documentation"
    text="The Agoric platform makes it possible to write safer smart contracts with your JavaScript skillset."
  />
  <HomeButtonRow
    title1="What's Agoric"
    text1="Learn about Agoric and its Javascript platform"
    link1="/guides/what-is-agoric/"
    title2="Getting Started"
    text2="Set up your environment and start building apps"
    link2="/guides/getting-started/"
    title3="Contract Framework"
    text3="Set up your environment and start building"
    link3="/guides/zoe/"
  />
  <HomeButtonRow

>     title1="Token Standards"
>     text1="Understand the ins and outs of Agoric's ERTP standard"

Reviewer: "ERTP standard" seems a little like ATM machine. ERTP is a protocol. Electronic Rights Transfer Protocol.

```suggestion
    title1="Token Protocol"
    text1="Understand the ins and outs of Agoric's Electronic Rights Transfer Protocol (ERTP)"
```

    link1="/guides/ertp/"
    title2="Samples"
    text2="Look through our smart contract and dapp examples"
    link2="/guides/zoe/contracts/"
    title3="Integrations"

>     text3="Browse through our catalogue of integrations"

Reviewer: hm. how about:

```suggestion
    text3="Integrate wallets etc. with Agoric like you would with other Cosmos SDK chains."
```

>     link3="/guides/chainlink-integration.html#overview"

Reviewer: chainlink? oh. weird. re-used the filename. odd.

    />
</div>

<div class="home-section">
  <HomeButtonHeader
    title="Blockchain resources"
    text="The tools your need to get the job done."
  />
  <HomeButtonRow
    title1="Block Explorer"
    text1="View transactions across the Agoric chain"
    link1="https://bigdipper.live/agoric"
    title2="Keplr Wallet"
    text2="A native wallet running with HardenedJS under the hood"
    link2="https://www.keplr.app/download"
    title3="Component Library"
    text3="Pre-built smart contracts for DeFi, NFTs, and cross-chain!"
    link3="https://components.agoric.com/"
  />
  <HomeButtonRow
    title1="Cosmos SDK"

>     text1="Our battle-tested consensus mechanism"

Reviewer: "Our" takes more credit than I'm comfortable with. Suggest:

```suggestion
    text1="The battle-tested consensus mechanism we build on"
```

    link1="https://docs.cosmos.network/"
    title2="IBC"

>     text2="The protocol ensuring Agoric is interoperable with 60÷ chains"

Reviewer: typo?

```suggestion
    text2="The protocol ensuring Agoric is interoperable with 60+ chains"
```

Author: OIC. Thank you!

    link2="https://ibc.cosmos.network/main"
    title3="Bounties"
    text3="A rotating list of incentivized bounties to grow our platform"

>     link3="https://components.agoric.com/bounties/open-bounties"

Reviewer: Placing "Bounties" here next to Cosmos and IBC seems odd.

I suggest swapping it with Integrations.

Author: Swapped

    />
</div>

<div class="home-section">
  <HomeButtonHeader
      title="Ready to learn more?"
      text="Once you've completed the Getting Started, here are a few next steps."
    />
  <h3 style="margin-top: 0em;">
    <a href="/guides/js-programming/hardened-js.html">Learn the basics of HardenedJS</a>
  </h3>

>   It's just JavaScript but with extra safety features - see why MetaMask uses it to safely run their Snaps technology for 30M+ customers.

Reviewer: I don't think "extra" is the way we describe it. It's more like: we removed the insecure stuff.

```suggestion
  It's just JavaScript but with hazardous stuff turned off - see why MetaMask uses it to safely run their Snaps technology for 30M+ customers.
```

But I suppose `harden()` _is_ an extra. Hm.
